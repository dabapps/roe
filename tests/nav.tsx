import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Nav from '../src/ts/components/navigation/nav';

describe('Nav', () => {

  it('should match snapshot', () => {
    const tree = renderer.create(<Nav />);

    expect(tree).toMatchSnapshot();
  });

  it('should accept regular element attributes', () => {
    const tree = renderer.create(<Nav className="my-class" />);

    expect(tree).toMatchSnapshot();
  });

});
