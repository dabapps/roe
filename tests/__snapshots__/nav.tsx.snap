// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should accept regular element attributes 1`] = `
<ul
  className="nav my-class"
/>
`;

exports[`Nav should match snapshot 1`] = `
<ul
  className="nav"
/>
`;
